build.gradle

```diff
@@ -15,7 +15,7 @@ sourceCompatibility = JavaVersion.VERSION_11
 
 allprojects {
 	group = "bio.terra"
-	version = "0.8.0-SNAPSHOT"
+	version = "0.9.0-SNAPSHOT"
```

Comment:
Why not 0.8.1 if it's a bugfix?

Reply:
Because we're changing an API in a way that's incompatible between versions. I don't really like putting a change like that in a bugfix version (though iiuc, versioning rules are generally much looser for major version 0)

Reply:
Sure, but it's not a breaking change unless it worked before. 😉